superset-frontend/src/components/FilterableTable/index.tsx

```diff
@@ -249,11 +257,16 @@ const FilterableTable = ({
       }),
   }));
 
+  const canExportData = useSelector((state: RootState) =>
+    findPermission('can_export_csv', 'SQLLab', state.user?.roles),
+  );
+
```

Comment:
The FilterableTable is a common component, so accessing the Redux user state to check permissions is not an appropriate approach. (Additionally, if FilterableTable is used outside of SQLLab, this permission logic would be incorrect.)
Therefore, it would be more appropriate to add a prop to FilterableTable (e.g., disableTextSelection?: boolean) and include this logic in SqlLab/components/ResultSet, passing it to FilterableTable.